connectors/spark/src/main/scala/io/unitycatalog/spark/utils/OptionsUtil.java

```diff
@@ -9,9 +9,6 @@ private OptionsUtil() {
   public static final String URI = "uri";
   public static final String TOKEN = "token";
   public static final String WAREHOUSE = "warehouse";
-  public static final String OAUTH_URI = "oauth.uri";
```

Comment:
Move from here to the https://github.com/unitycatalog/unitycatalog/pull/1261/files#diff-62ea646a74fab831be8a8d08ff4a3086e2de16fb4f2f9d344617bc39bd8e9185R3, which make more sense because those properties are used for unitycatalog client authentication.